README.md

```diff
@@ -73,3 +75,13 @@ The server will start locally and on ngrok. The restaurants will not load becaus
 
 * [Here is the City's list of layers](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers), each of which provides a REST endpoint from which we can query data. 
 * [Here's an example of a URL where you might query data](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers/ParkLocations/MapServer/0/query).
+
+# Developers
+
+Instead of building the server and launching the server via make run, an alternative method to start the server is:
+>export GOOGLE_PLACES_API_KEY=supersecretkey
+>cd Frontier
+>npm start
+
+This eliminates the required build step allowing for faster debug cycles.
```

Comment:
Specifically, this avoids having to build the whole docker image and instead simply runs the project.  

     npm start

is still running webpack which is technically a build step.